front/app/components/EsriMap/index.tsx

```diff
@@ -65,13 +63,8 @@ const EsriMap = ({
     });
 
     // Set the basemap
-    const webTileLayerFromUrl = new WebTileLayer({
-      urlTemplate: globalMapSettings?.tile_provider || DEFAULT_TILE_PROVIDER,
-      copyright: getTileAttribution(globalMapSettings?.tile_provider || ''),
-    });
-
     esriMap.basemap = new Basemap({
-      baseLayers: [webTileLayerFromUrl],
+      baseLayers: [getDefaultBasemap(globalMapSettings?.tile_provider)],
```

Comment:
The changes here are just from moving the fetching of the map basemap from this file into the util file.